docs/clients/ecr.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: client
+category: clients
+name: ECR
+package: async-aws/ecr
+---
+
+## Usage
+
+### Invoke a function
```

Comment:
@Nyholm this is wrong. copied from IAM which was copied from Lambda

Reply:
Oops. Can you fix that in IAM and ECR?